crates/rolldown/src/stages/generate_stage/mod.rs

```diff
@@ -227,7 +229,7 @@ impl<'a> GenerateStage<'a> {
 
     let mut hash_placeholder_generator = HashPlaceholderGenerator::default();
 
-    let mut used_name_counts = FxHashMap::default();
+    let used_name_counts = FxDashMap::default();
```

Comment:
Why it requres `DashMap`?

Reply:
The assets assigned names required it. So here changed it.